app/src/main/java/com/example/cristiana/workshop1/model/Repository.java

```diff
@@ -0,0 +1,131 @@
+package com.example.cristiana.workshop1.model;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by Cristiana on 4/4/2017.
+ */
+
+public class Repository {
+    String mName;
+    String mOwner;
+    String mDescription;
+    boolean mIsPrivate;
+    String mDefaultBranch;
+    List<String> mTopics;
+    int mWatchersCount;
+    Date mCreatedAt;
+    Date mUpdatedAt;
+
+    public static final List<Repository>sMockRepository;
+
+    public Repository(String mName, String mOwner, String mDescription, boolean mIsPrivate,
```

Comment:
Poti specifica in Android Studio (Preferences -> Code Style -> Java):
Fields sa aiba prefixul "m" si static fields sa aiba prefixul "s" -> Apply

Astfel cand iti va genera constructor + getters/setters sa tina cont de prefix si sa nu ajungi cu prefixul si in numele argumentelor sau al getterilor/setterilor

vezi urmatorul link: https://i.stack.imgur.com/g55ba.png